### DISCUSSION TITLE
Failed to load resources from server (when deployed on Google Cloud) - net::ERR_ABORTED 404

### DISCUSSION THREAD

**Discussion by siowzhenhan** [vercel/next.js#23268] [Help]
### What version of Next.js are you using?

10.0.8

### What version of Node.js are you using?

12.13.0

### What browser are you using?

Chrome

### What operating system are you using?

MacOS

### How are you deploying your application?

next build && gcloud functions deploy

### Describe the Bug

Failed to load ANY resources from the Nextjs server after deployment to Google cloud functions. However everything works when project is run locally, i.e. using next dev or next start after running next build.

### Expected Behavior

Expected the project to run as normal as on `next dev` or `next start`.

### To Reproduce

My next.config.js:
 ``` NODE_ENV,
  NEXT_PUBLIC_BASE_PATH,
  NEXT_PUBLIC_FRONTEND_URL
} = process.env
const isProd = NODE_ENV === 'production'

module.exports = {
  target: 'serverless',
  basePath: NEXT_PUBLIC_BASE_PATH,
  assetPrefix: isProd ? NEXT_PUBLIC_BASE_PATH : '',
  publicRuntimeConfig: {
    basePath: NEXT_PUBLIC_BASE_PATH || ''
  }
}
```
This is the error logged on the browser console:

![Screenshot 2021-03-22 at 12 37 17 PM](https://user-images.githubusercontent.com/61175015/111960417-afd82f00-8b2a-11eb-9f9c-1c48af759c0b.png)

**Comment by rajkissu** [comment]
Experiencing the same problem here too - works fine on local, but does not work when pushed to Google Cloud Functions.

I followed instructions here https://daveteu.medium.com/nextjs-with-google-cloud-functions-in-10-minutes-47f88e6ac27a and added `basePath` since Google Cloud Functions works off a subpath (no subdomain/domain mapping as far as I can see) and the URLs get messed up without it when deployed, but I am getting a lot of 404 errors that prevent the website from loading.

Any idea what is the cause?
